common/src/main/java/org/mcaccess/minecraftaccess/mixin/GameNarratorAccessor.java

```diff
@@ -0,0 +1,11 @@
+package org.mcaccess.minecraftaccess.mixin;
+
+import net.minecraft.client.GameNarrator;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(GameNarrator.class)
+public interface GameNarratorAccessor {
+    @Invoker("narrateMessage")
```

Comment:
This argument is unnecessary, it's derived from the method name if it's omitted.
```suggestion
    @Invoker
```